regression_tests/operator.go

```diff
@@ -0,0 +1,262 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"os/exec"
+	"runtime"
+	"strings"
+	"time"
+
+	walletclient "github.com/bitcoin-sv/spv-wallet-go-client"
+	"github.com/bitcoin-sv/spv-wallet-go-client/xpriv"
+)
+
+const (
+	domainLocalHost     = "http://localhost:3003"
+	adminXPriv          = "xprv9s21ZrQH143K3CbJXirfrtpLvhT3Vgusdo8coBritQ3rcS7Jy7sxWhatuxG5h2y1Cqj8FKmPp69536gmjYRpfga2MJdsGyBsnB12E19CESK"
+	adminXPub           = "xpub661MyMwAqRbcFgfmdkPgE2m5UjHXu9dj124DbaGLSjaqVESTWfCD4VuNmEbVPkbYLCkykwVZvmA8Pbf8884TQr1FgdG2nPoHR8aB36YdDQh"
+	leaderPaymailAlias  = "leader"
+	minimalBalance      = 100
+	defaultGoClientPath = "../../spv-wallet-go-client/regression_tests"
+	defaultJSClientPath = "../../spv-wallet-js-client/regression_tests"
+)
+
+func main() {
+	loadConfigFlag := flag.Bool("l", false, "Load configuration from .env.config file")
+	flag.Parse()
+
+	var config *regressionTestConfig
+	var err error
+	user := &regressionTestUser{}
+
+	if *loadConfigFlag {
+		config, err = loadConfig()
+		if err != nil {
+			fmt.Println("error loading config:", err)
+			return
+		}
+		user.XPriv = config.ClientOneLeaderXPriv
+	} else {
+		config = &regressionTestConfig{}
+	}
```

Comment:
Consider making a utility function for that part of code :)

Reply:
After DM conversation - we'll maybe do it in the second iteration

Reply:
Changed to unresolved as more changes were needed. 
Utility function made.